styles.css

```diff
@@ -0,0 +1,278 @@
+    color: white;
+}
+
+.awesome {
+    font-family: 'Raleway', sans-serif;
+    width: 1170px;
+    margin: 0 auto;
+}
+
+.awesome__title, .price__title, .customers__title {
+    font-size: 36px;
+    font-weight: 400;
+}
+
+.awesome__title-wrapper, .price__title-wrapper, .customers__title-wrapper {
+    height: 150px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 50px;
+}
+
+.awesome__line, .price__line, .customers__line {
+    width: 170px;
+    border: 1px solid #EC7979;
+    margin: auto;
+    margin-bottom: 40px;
+}
+
+.awesome__subtitle {
+    color:#86878B;
+    font-size: 18px;
+    text-align: center;
+    margin-bottom: 110px;
+}
+
+.awesome__container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 58px;
+}
+
+.awesome__block {
+    width: 370px;
+    text-align: center;
+}
+
+.awesome__icon-wrapper {
+    width: 92px;
+    height: 92px;
+    border: 1px solid #EC7979;
+    border-radius: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto 78px;
+}
+
+.awesome__block-title {
+    font-size: 24px;
+    font-family: 'Open Sans', sans-serif;
+    text-align: center;
+    margin-bottom: 47px;
+}
+
+.price {
+    width: 1508px;
+    margin: 0 auto;
+    background-color: #F5F5F5;
+    font-family: 'Raleway', sans-serif;
+}
+
+.price__subtitle {
+    color:#86878B;
+    font-size: 18px;
+    text-align: center;
+    margin-bottom: 79px;
+}
+
+.price__container {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 170px 0 168px;
+    padding-bottom: 50px;
+}
+
+.price__block, .customers__block {
+    width: 370px;
+}
+
+.price__block:hover {
+    filter: drop-shadow(0px 4px 10px rgba(51, 51, 51, 0.2));
+    margin-top: -20px;
+}
+
+.price__price-title {
+    height: 60px;
+    font-family: 'Open Sans';
+    font-size: 24px;
+    font-weight: bold;
+    color: #FFFFFF;
+    background-color: #333333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid #F5F5F5;
+}
+
+.price__prices {
+    height: 100px;
+    font-size: 36px;
+    font-weight: bold;
+    color: #FFFFFF;
+    background-color: #333333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid #F5F5F5;
+    margin-top: -1px;
+}
+
+.price__price-subtitle {
+    height: 50px;
+    font-family: 'Raleway';
+    font-size: 18px;
+    font-weight: 400;
+    color: rgba(255, 255, 255, 0.5);
+    background-color: #333333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid #F5F5F5;
+    margin-top: -1px;
+}
+
+.price__list {
+    height: 50px;
+    font-family: 'Raleway';
+    font-size: 18px;
+    font-weight: 400;
+    color: #86878B;;
+    background-color: #FFFFFF;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 2px solid #F5F5F5;
+    margin-top: -1px;
+}
```

Comment:
Code duplicating. Do like you've done in 84 line. Common styles, then separately different styles.